### DISCUSSION TITLE
Console error: ALPINE_VERSION is not defined (since v3.12.3)

### DISCUSSION THREAD

**Discussion by allmarkedup** [alpinejs/alpine#3855] [7. Bugs (archived)]
I've just tried updating Alpine to the latest release and i'm seeing an error in the console that ' ALPINE_VERSION is not defined' and Alpine is not working at all.

<img width="398" alt="CleanShot 2023-11-06 at 10 28 15@2x" src="https://github.com/alpinejs/alpine/assets/126726/b3b369d8-1cb6-4705-aff2-2077b7bd0997">

After some investigation it seems that I'm getting this same error from version 3.12.3 onwards. v3.12.2 is fine.

I'm bundling with esbuild and transpiling to ES2017.

Has anyone else seen the same error? Any clues as to what I can do to fix it? Any help would be much appreciated!

**Comment by ekwoka** [comment]
Can you give more info? The current version doesn't have that in the package on npm.

**Comment by allmarkedup** [comment]
Many apologies, my bad - I had forgotten I had imported a function from the Alpine `src` directory directly - it was this import that was throwing the error. It was no longer needed anyway so I've just removed that and all is good.

Thanks @ekwoka for the reply and sorry for wasting time!
